### ISSUE TITLE
Player character can arrange own marriage before Numenorean blood allows it

### ISSUE THREAD

**Issue by Lewa263** [jj248/RealmsInExile#1099]
When playing as a character who is not yet considered nubile for their Numenorean blood, the Arrange Marriage interaction is available, and marriage will happen if the targeted spouse is considered nubile. Marriages cannot be similarly arranged early for player family members, only the character being played. My best guess is that it has to do with being both the matchmaker and the actor in the `arrange_marriage_interaction`.

In this first screenshot, I had arranged a betrothal as this woman's father, and was able to convert it to a marriage at age 19 as soon as she inherited, even though her tier 1 Numenorean blood requires age 20 for marriage.
![2024_04_03_2](https://github.com/jj248/RealmsInExile/assets/18605346/967a8bc8-8901-4731-93cb-52f4549e6518)

This second screenshot shows that the betrothal part was not necessary. Player Boromir can marry from day 1 despite having level 2 Numenorean blood that should require age 25 for marriage.
![2024_04_03_3](https://github.com/jj248/RealmsInExile/assets/18605346/97e18d9b-38ae-4d12-95a9-3140ce0ce6c6)

**Comment by BriceLeduc** [comment]
Yes, that is intended. The Numenóreans are adult at 16. They don't usually seek marriage before later but a player can

**Comment by BriceLeduc** [comment]
It says so in the tooltip
